src/App.tsx

```diff
@@ -44,6 +53,16 @@ const App = () => {
 
   const classes = useStyles();
 
+  let player: string = !turn ? 'X' : 'O';
+
+  const handleCellValue = (idx: number): void => {
+    let cell = [...board];
```

Comment:
Since you're destructuring board and assigning it to this variable, it's not logical to name it as `cell`, since you're using it to store a board buffer you could call it like that: `boardBuffer`, also since you're not reassigning it, and it's an array you could turn it into a const.